README.md

```diff
@@ -1,25 +1,10 @@
-### Deploy a Kubernetes Cluster
+## Deploy a Kubernetes Cluster with Terraform
+This repository provides examples of provisioning Kubernetes clusters on major cloud providers using Terraform. These should be treated as illustrative examples, not production-ready infrastructure configuration.
 
-Pick the platform you want to deploy your cluster to, and follow the instructions in the README of the corresponding directory.
+### Get Started
+- Install [terraform](https://developer.hashicorp.com/terraform/downloads)
+- `cd` into the directory corresponding to your cloud provider (AWS, GCP, or Azure)
+- Follow the instructions in the corresponding README
```

Comment:
"Follow the instructions in the README in that directory"

Reply:
Pushed this exact change to main.